src/Client.php

```diff
@@ -15,19 +15,24 @@
 
 use Http\Client\Common\HttpMethodsClient;
 use Http\Client\Common\Plugin\BaseUriPlugin;
+use Http\Client\Common\Plugin\HeaderAppendPlugin;
 use Http\Client\Common\PluginClient;
 use Http\Client\Exception;
 use Http\Client\HttpClient;
 use Http\Discovery\HttpClientDiscovery;
 use Http\Discovery\MessageFactoryDiscovery;
 use Http\Discovery\UriFactoryDiscovery;
+use Nexy\Slack\Exception\SlackErrorException;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Psr\Http\Message\ResponseInterface;
 
 /**
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
 final class Client
 {
+    const SLACK_POST_MESSAGE_URL = 'https://slack.com/api/chat.postMessage';
```

Comment:
So the API structure is the same as web-hook but with a common URL now, am I right?

Reply:
Yep